dist/vue-mq.es.js

```diff
@@ -79,13 +89,15 @@ function isArray(arg) {
 }
 
 // USAGE
-// mq-layout(mq="lg")
-//   p I’m lg
 var component = {
   props: {
     mq: {
       required: true,
       type: [String, Array]
+    },
+    createContainer: {
+      type: Boolean,
+      default: false
```

Comment:
Should the default remain true since this could break some CSS? I agree that default should've been false all along, but I wonder if the new option has to lean towards precedence.

Reply:
Keep it true, for this pull request. so we do not break something for anyone.
this option is needed anyway. regardless of default-setting.

Reply:
Hi, this is great, but how do you use it ?